src/firmware.cpp

```diff
@@ -133,6 +133,43 @@ static CommandResponse servo(int requestID, String argument) {
   return OK;
 }
 
+static CommandResponse ultrasoundRawTiming(int requestID, String argument){
```

Comment:
other ultrasound function should call this function

Reply:
it was decided to create an IO function that does the triggering and that would be called by both of the US functions.